data/contributors.json

```
@@ -7,4 +7,8 @@
         "name": "Vinicius Barbosa",
         "github_user": "viniciusbds"
     }
```

@anacarolinacv tudo bom? Obrigada por contribuir aqui!

Tu poderia adicionar uma virgula aqui? Sem ela a estrutura do arquivo é quebrada
```suggestion
    },
```